pkg/model/manifest.go

```diff
@@ -682,7 +682,7 @@ func (m *Manifest) SanitizeSvcNames() error {
 			}
 			m.Build[sanitizedSvcName] = build
 			sanitizedServicesNames[buildKey] = sanitizedSvcName
-			delete(m.Dev, buildKey)
+			delete(m.Build, buildKey)
```

Comment:
Can we include some tests to this function? Checking the code it shouldn't be hard, right? if it has already some test, can we include this scenario?